### DISCUSSION TITLE
SVG overlay rendering issues in Chrome 105

### DISCUSSION THREAD

**Discussion by matt-farmer** [Leaflet/Leaflet#8448] [General]
Just wanted to raise this for information, as it's not really a Leaflet issue, but others may  be experiencing the same issues as us with the new 105 release of Chrome.

We use svg image overlays on leaflet, and since the release of Chrome 105 overlays, particularly those with a lot of text elements, are rendering very slowly. Panning and zooming are particularly bad. 

This hasn't been an issue on Chrome previously, and performance on Safari is still smooth as ever.

I've downloaded the latest Chrome Beta, and all the issues have gone away, rendering is back to the previous levels of pretty much instant, and panning zooming all smooth as before.

Hopefully this means the issue will be resolved when the next Chrome update comes out, but thought I'd raise it here in case others are finding unexpected poor behaviour suddenly in Chrome, and (like us) wondering what you might have done wrong!

I have raised a support issue with Chrome to make them aware, but looks like 105 had a lot of performance issues across the board, so expect behaviour to be back to normal with the next release.

**Comment by anton-seaice** [comment]
Hi Matt. Have you found this issue resolved? I still find panning and zooming with lots of SVG labels to be slow on Chrome 109 and also Edge.

**Comment by matt-farmer** [comment]
Hey Anton, for now yes our performance is ok (although far better on recent Safari builds). We might get up to perhaps several hundred labels on an svg, not sure how many you would have. I will say, though, that i've had to learn more about the intricacies of creating efficient svgs than I ever imagined, so please feel free to contact me directly if I can pass any of that knowledge on in case it helps!
